src/routes/donate/TeamMember.svelte

```diff
@@ -28,21 +28,23 @@
 				<h4>{member.name}</h4>
 			</a>
 			<div class="verified-badge">
-				<ToolTip
-					content="<p>GPG key ID:</p> <a class='gpg-url' href={member.gpg_key
-						.url} rel='noreferrer' target='_blank'>{member.gpg_key.id}</a>"
-					html={true}
-				>
-					<div class="desktop">
-						<a href={member.gpg_key.url} rel="noreferrer" target="_blank">
+				{#if member.gpg_key}
```

Comment:
The badge would still show up even without a gpg key. This is not correct behavior

Reply:
Oops. My bad, was an oversight. Now it should be correct, I should have grouped the entire badge element.